src/components/AddBook.js

```diff
@@ -0,0 +1,15 @@
+const AddBook = () => (
+  <>
+    <h1>ADD NEW BOOK</h1>
+    <form>
+      <input
+        type="text"
+      />
+      <input
+        type="text"
+      />

```

Comment:
- [**Optional**] It's really difficult for me to identify fields for title and author :angry:. You may resolve this issue by adding a placeholder.